File: DSA/SakshamDwivedi_590013147/Day2/D2P1.c
#include <stdio.h>

int arr[100];
int size;

int main(){
    //int arr[10] = {1, 2, 3, 4, 5, 6};
    printf("Enter the size of array : ");
    scanf("%d", &size);
    printf("Enter the elements of array : ");
    for(int i = 0; i < size; i++){
        scanf("%d", &arr[i]);
    }
    int result = 0;

    for(int i = 0; i < size; i++){
        if(arr[i] % 2 == 0){
            result += arr[i];
        }
    }

    printf("Sum : %d", result);

    return 0;
}

File: DSA/SakshamDwivedi_590013147/Day2/D2P1.py
#Fixed Array
arr = [1, 2, 3, 4, 5]
result = 0
for i in range(len(arr)):
    if arr[i] % 2 == 0:
        result += arr[i]
print("Sum : ", result)

#Custom Input Array
arr = list(int(x) for x in input("Enter the array : ").split())
result = 0
for i in range(len(arr)):
    if arr[i] % 2 == 0:
        result += arr[i]
print("Sum : ", result)
